evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -38,10 +38,10 @@
 # values for approval states shown to staff
 StateValues = namedtuple('StateValues', ('order', 'icon', 'filter', 'description'))
 APPROVAL_STATES = {
-    'new': StateValues(0, 'fas fa-circle icon-yellow', 'fa-circle icon-yellow', _('In preparation')),
-    'prepared': StateValues(2, 'far fa-square icon-gray', 'fa-square icon-gray', _('Awaiting editor review')),
-    'editor_approved': StateValues(1, 'far fa-check-square icon-yellow', 'fa-check-square icon-yellow', _('Approved by editor, awaiting manager review')),
-    'approved': StateValues(3, 'far fa-check-square icon-green', 'fa-check-square icon-green', _('Approved by manager')),
+    'new': StateValues(0, 'fas fa-circle icon-yellow', 'new', _('In preparation')),
+    'prepared': StateValues(2, 'far fa-square icon-gray', 'prepared', _('Awaiting editor review')),
+    'editor_approved': StateValues(1, 'far fa-check-square icon-yellow', 'editor_approved', _('Approved by editor, awaiting manager review')),
+    'approved': StateValues(3, 'far fa-check-square icon-green', 'approved', _('Approved by manager')),
```

Comment:
I can't find the place where those are used, so: why does it help to have the key as part of the value tuple?

Reply:
Actually, it is well-hidden behind the `|approval_state_values` template filter. The property `filter` is used here:
https://github.com/fsr-de/EvaP/blob/f713938529166613866996222d657cead2ff3322/evap/staff/templates/staff_semester_view_evaluation.html#L18
One might ask, why not use the state name as-is? Well, we want to clamp all higher states to `approved`. This could be implemented differently, but I just kept the structure like I found it